### PR TITLE
chore: add code reviewers github action

### PR DESCRIPTION
**Pull request by richbachman**
## Description of the change

Added an action to assign specific PR reviewers. Also trying to solve the Kodiak auto-merging.

## Testing the change

- [ ] Write your testing instructions here

## Type of change

- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Non-Breaking Change (change to existing functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### Development

- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development

### Code review

- [ ] This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached.

### CODE REVIEW

File: .github/reviewers.yml
Comment (blumaa) [inline .github/reviewers.yml:8]: lol Brian has a legitimate name. :)

**Review by blumaa — APPROVED** [review]

**Review by RohitRajendran — COMMENTED** [review]
Do you need the action? Couldn't you just list the users as codeowners instead?

File: .github/reviewers.yml
Comment (apvale) [inline .github/reviewers.yml]: Shouldn't we set number of approvers as well? 🤔

File: .kodiak.toml
Comment (apvale) [inline .kodiak.toml:4]: Me again about this, Rich 😅 
If the label is being used in the PR when will the PR be merged? When it has 1 approval?

**Review by RohitRajendran — APPROVED** [review]
👍🏽

File: .kodiak.toml
Comment (richbachman) [reply .kodiak.toml:4]: PRs now require 2 approvals. Once all checks pass, and there are two approvals it should auto-merge.

File: .github/reviewers.yml
Comment (apvale) [reply .github/reviewers.yml]: But I think this is a conversation about the process in Pluto 😅  We still don't know how many approvals we want to have by PR. 🙏🏻

Comment (richbachman) [reply .github/reviewers.yml]: At our current team size, I feel like 2 approvals will be plenty. Any design changes will also need the Chromatic approval. We can re-evaluate if we find things are slipping through.

**Pull request merged**
